frontend/routes/ConfigRoute.tsx

```diff
@@ -1,16 +1,23 @@
 import * as React from 'react';
-import { BreadcrumbActions, GameVisibilityActions } from '../storage/common';
+import axios from 'axios';
+import { BreadcrumbActions, GameVisibilityActions, SERVER_ADRESS, DownloadActions, AppActions, AppStore } from '../storage/common';
 import { connect } from 'react-redux';
 import { Dispatch, AnyAction } from 'redux';
 import { ConfigView } from '../views/ConfigView';
+import { Game } from '../storage/models/Game';
 
 interface ConfigRouteProps {
+	games: Map<number, Game>;
 	dispatch: Dispatch<AnyAction>;
 }
 
 export class ConfigRoute extends React.Component<ConfigRouteProps> {
 	public componentDidMount() {
 		this.props.dispatch(BreadcrumbActions.setBreadcrumb('Config'));
+
+		// if (this.props.games.size === 0) {
+		fetchGameData()(this.props.dispatch);
+		// }
 	}
 	public render() {
 		const changeGamesVisibilityCallback = (allVisible: boolean) => {
```

Comment:
If I understand correctly, what will happen at the moment is:
1. You go to the config view page
2. It displays empty list of games for a short while
3. Once the games are fetched the component refreshes with the actual data?

Reply:
2. is a little bit different, because it has placeholder data from `createInitialApplicationState`, but yeah. 
When you stated it it looks pretty inelegant as a solution.

Reply:
That's okay, we can improve it in a later step :)